### DISCUSSION TITLE
Enter Reading View after creating note

### DISCUSSION THREAD

**Discussion by gmakstutis** [SilentVoid13/Templater#484] [Help]
Hello,

I'm looking for a way to make Obsidian enter 'Reading View' (preview) after a new note is created using templater. When I have a new note created through executing the templater commands, the note appears in the editing mode, but I'd like it to switch to 'Reading View' automatically.

Is this possible?

Thanks

**Comment by agnoldo** [comment]
Hello,

I see two ways that can be tested:

- changing the "Default new pane view" (to Reading View, in Settings - Editor). But this will change for all notes
- using the "Force note view mode" plugin (you just add a YAML field to your template)

HTH!

**Comment by shabegom** [comment]
You could add this as a templater command:

```
<%*
let state = app.workspace.activeLeaf.getViewState()
state.state.mode = "preview"
await app.workspace.activeLeaf.setViewState(state)
-%>
```
